FirebaseFirestoreInternal/FirebaseFirestore/FIRPipelineBridge.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <FirebaseFirestoreInternal/FIRPipelineBridge.h>
```

Comment:
The goal of these redirects was to preserve client header imports (as opposed to module imports) when `FirebaseFirestore` became a Swift module. IUUC, no ObjC clients are expected to import `FIRPipelineBridge`, so this file is not needed.